ihmc-avatar-interfaces/src/main/java/us/ihmc/avatar/networkProcessor/kinemtaticsStreamingToolboxModule/KinematicsStreamingToolboxParameters.java

```diff
@@ -293,7 +304,7 @@ public void setDefault()
       useStreamingPublisher = true;
       publishingPeriod = 5.0 * 0.006;
 
-      inputStateEstimatorType = InputStateEstimatorType.FIRST_ORDER_LPF;
+      inputStateEstimatorType = InputStateEstimatorType.FBC_STYLE;
```

Comment:
The default value shouldn't be changed, I still want by default the old state estimator. Only change the estimator in your test and in `NadiaKinematicsStreamingToolboxModule`